TypeCobol/Compiler/Sql/Model/FromClause.cs

```diff
@@ -1,10 +1,16 @@
-﻿namespace TypeCobol.Compiler.Sql.Model
+﻿using System.Collections.Generic;
+using TypeCobol.Compiler.CodeElements;
+
+namespace TypeCobol.Compiler.Sql.Model
 {
     public class FromClause : SqlObject
     {
-        //TODO
-        //list of table references
-
+        public List<SymbolReference> ViewReferences { get;}
```

Comment:
These are either tables or views so the naming is incorrect. `TableOrViewReferences`